build.py

```diff
@@ -154,6 +204,31 @@ def get_package_type():
     log.debug('package type detected: {0}'.format(pt))
     return pt
 
+def get_package_dependencies(v):
+    log = logging.getLogger(__name__)
+    deps = []
+
+    # dependencies on other externals packages
+    if 'interdependencies' in v:
+        for d in v['interdependencies']:
+            deps.append(get_package_name(d))
+
+    # distro-specific dependencies
+    if 'distro_dependencies' in v:
+        distro_type = get_distribution_type()
+        v_distro_deps = v['distro_dependencies']
+        if distro_type in v_distro_deps:
+            distro_ver = get_distribution_version()
+            distro_type_deps = v_distro_deps[distro_type]
+            if distro_ver in distro_type_deps:
+                deps.extend(distro_type_deps[distro_ver])
+            else:
+                log.debug('version %s of distro %s not in distro_dependencies', distro_ver, distro_type)
+        else:
+            log.debug('distro %s not in distro_dependencies', distro_type)
```

Comment:
Please wrap the placeholders in square brackets so it's obvious what the replacement arg is. Helps with identifying empty strings and whatnot.